test/test_dht.cpp

```diff
@@ -2692,6 +2704,39 @@ TORRENT_TEST(node_id_bucket_distribution)
 	}
 }
 
+TORRENT_TEST(node_id_min_distance_exp)
+{
+	node_id n1 = to_hash("0000000000000000000000000000000000000002");
+	node_id n2 = to_hash("0000000000000000000000000000000000000004");
+	node_id n3 = to_hash("0000000000000000000000000000000000000008");
```

Comment:
I think it's nice to declare n1, n2 and n3 as const. (and I prefer "continental const placement", i.e. to the right of the type it applies to: "node_id const n1 =")